src/main/java/v/hryhoryk/onlinebookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,14 +1,27 @@
 package v.hryhoryk.onlinebookstore.dto;
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Pattern;
+import jakarta.validation.constraints.Positive;
 import java.math.BigDecimal;
+import org.hibernate.validator.constraints.URL;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Why do you need this annotation?

Reply:
I actually thought that we are not specifying id in our DTO and there will be some kind of conflict while parsing this file by JSON, but now I've read documentation properly and understood that it's unnecessary here